EIPS/eip-7517.md

```diff
@@ -0,0 +1,116 @@
+* `aiTraining`: Allow the asset to be used as training data for generative and non-generative AI/ML models.
+* `aiTrainingWithAuthorship`: Same as `aiTraining`, but requires that the authorship is disclosed.
+
+### Schema
+
+```json
+{
+    "type": "object",
+    "properties": {
+        "allowed": {
+            "enum": [ "dataMining", "aiTraining", "aiTrainingWithAuthorship", "aiGenerativeTraining", "aiGenerativeTrainingWithAuthorship", "aiInference" ],
+            "description": "The asset may be used for training an AI/ML model or mined for its data (or both)."
+        },
+        "constrainted": {
+            "enum": [ "dataMining", "aiTraining", "aiTrainingWithAuthorship", "aiGenerativeTraining", "aiGenerativeTrainingWithAuthorship", "aiInference" ],
+            "description": "The asset may be used for training an AI/ML model or mined for its data (or both) followed the license."
+        },
+        "notAllowed": {
+            "enum": [ "dataMining", "aiTraining", "aiTrainingWithAuthorship", "aiGenerativeTraining", "aiGenerativeTrainingWithAuthorship", "aiInference" ],
+            "description": "The asset may not be used for training an AI/ML model or mined for its data (or both)."
+        }
+    },
+    "required": [ "allowed", "constrainted", "notAllowed" ]
+}
+```
+
+### Examples
+
+The mining preference example for not allowing generative AI training:
+
+```json
+{
+    "miningPreference": {
+        "allowed": [ "dataMining", "aiTrainingWithAuthorship", "aiInference" ],
+        "contrained": [],
+        "notAllowed": [ "aiGenerativeTraining" ]
+    }
+}
+```
+
+The mining preference example for only allowing for AI inference:
+
+```json
+{
+    "miningPreference": {
+        "allowed": [ "aiInference" ],
+        "contrained": [],
+        "notAllowed": [ "aiTraining", "aiGenerativeTraining" ]
+    }
+}
+```
+
+The mining preference example for allowing generative AI training if mentioning authorship and follow license:
+
+```json
+{
+    "miningPreference": {
+        "allowed": [ "dataMining", "aiTrainingWithAuthorship", "aiInference" ],
+        "contrained": [ "aiGenerativeTrainingWithAuthorship" ],
+        "notAllowed": []
+    }
+}
+```
+
+## Rationale
+
+The introduction of the `miningPreference` property in digital asset metadata covers the considerations including
+
+Accessibility: A clear and easily accessible method with human-readibility and machine-readibility for digital asset creators and license owners to express their preferences for how their assets are used in data mining and AI/ML training workflows. The AI/ML creators can check and collect data systematically.
+Adoption: As Coalition for Content Provenance and Authenticity (C2PA) already outlines guidelines for indicating whether an asset may be used in data mining or AI/ML training, it's crucial that onchain metadata aligns with these standards. This ensures compatibility between in-media metadata and onchain records.
```

Comment:
This all reads like Motivation. The Rationale section should justify specific technical choices made within the proposal itself (eg. why you differentiate different types of training, or why your schema permits having the same permission in both allowed and notAllowed). The Motivation section justifies the EIP as a whole.

Reply:
Also, it's worth noting that C2PA is very flawed--it's been promoted as a digital 'proof of truth', while it's actually just a centralized web of trust that allows governments to have control over what is digitally considered 'true', and the ability to track individuals by which files they edit and share. Its standard is straight up incompatible with Ethereum; I'd highly recommend using another source (this is not a requirement; just a recommendation).

Reply:
Please remember that in almost every jurisdiction nowadays, a person has the right to their personal data. They may decide to revoke consent to process it, may request their personal data/data porting, may ask for correcting false data or to erase any personal data (e.g. under GDPR/EU law or most cyber law). I myself would never allow for my personal data, out of blockchain data collections for example, to be processed by AI/ML data mining, so not sure this is really fashionable. Just saying…